syntax_and_semantics/annotations.md

```diff
@@ -1,6 +1,6 @@
-# Attributes
+# Annotations
 
-Some types and methods can be annotated with attributes. The attribute list is fixed, but eventually (maybe) there will be user-defined attributes.
+Some types and methods can be annotated. 
```

Comment:
```suggestion
Some types and methods can be annotated with inbuilt annotations and user-defined annotations.
```

Reply:
I don't think "inbuilt" is the right word here. Maybe "builtin" or "compiler-defined"?

Reply:
Not a big fan of `inbuilt`, would rather have `can be annotated with build-in annotations and ...`

Reply:
```suggestion
Some types and methods can be annotated with inbuilt annotations and user-defined annotations.
```